### PR TITLE
[SYSSETUP] Add en-GB.rc

### PR DESCRIPTION
**Pull request by PopuriAO29**
The British Language wasn't included here. However, i have corrected some texts that are still American English.

## Purpose

The British English resource wasn't exist while during Setup. however, There might include various American English text including the word "Customize", So i have decided add these on it.

## Proposed changes

_Describe what you propose to change/add/fix with this pull request._

- The file en-GB.rc isn't exist on SYSSETUP Language Resource folder.

### CODE REVIEW

File: dll/win32/syssetup/lang/en-GB.rc [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
```
Comment (binarymaster) [inline dll/win32/syssetup/lang/en-GB.rc:19]: Also you can just drop this header.

**Review by binarymaster — CHANGES_REQUESTED** [review]
Please add created `en-GB.rc` file to the main resource source here: https://github.com/reactos/reactos/blob/master/dll/win32/syssetup/syssetup.rc#L65

**Review by binarymaster — APPROVED** [review]
Ok I also noticed PR you created in your own repo https://github.com/PopuriAO29/reactos/pull/1 - I think it's highly related and it should be done here as well, so I committed it to the branch of this PR.

**Comment by eason329** [comment]
Actually what is the purpose of changing the translation for language files other than English? (Because it made conflicts to my PR.)

**Comment by binarymaster** [comment]
There was a typo in english translation, but some other translation files were untranslated, so they contained the same typo.

**Pull request merged**